src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -189,14 +190,14 @@ def apply_quantization_config(
     return names_to_scheme
 
 
-def process_quantization_config(config: QuantizationConfig) -> QuantizationConfig:
+def process_quantization_config(config: Optional[QuantizationConfig]) -> Optional[QuantizationConfig]:
     """
     Preprocess the raw QuantizationConfig
 
-    :param config: the raw QuantizationConfig
-    :return: the processed QuantizationConfig
+    :param config: Optional raw QuantizationConfig
+    :return: the processed QuantizationConfig, if the raw config is not None
     """
-    if config.kv_cache_scheme is not None:
+    if config is not None and config.kv_cache_scheme is not None:
```

Comment:
Do we use this function outside of `apply_quantization_config`? If not, we would never hit the `None` case?